app/resources/css-default.template

```diff
@@ -177,11 +177,16 @@ div.languages {
   text-decoration: none;
   color: black;
   text-align: {{end}};
+  float:right;
+  margin-right:5%;
```

Comment:
Thanks for the finding. It looks related to #351 which I have just filed. Looks like the whole .header rule in sidebar.css is not necessary. It should be fixed together with #351.